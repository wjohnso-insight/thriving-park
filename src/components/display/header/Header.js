//@flow
import * as React from 'react'
import { Link } from "react-router-dom"
import "./Header.scoped.scss"

type Props = {

}

export default function Header(props: Props) : React.Node {
    return (
        <section id="header-wrapper">
            <section>
                <h1>
                    Driving Park
                </h1>
            </section>
            <nav id="header-nav">
                <Link to="/">ABOUT</Link>
                <Link to="/move">MOVE</Link>
                <Link to="/litter">LITTER</Link>
            </nav>
        </section>
    )
}