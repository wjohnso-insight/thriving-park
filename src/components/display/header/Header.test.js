import React from 'react'
import Header from './Header'
import { render } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom'

it('renders the component', () =>{
    const container = render(
        <BrowserRouter>
            <Header />
        </BrowserRouter>
    )
    // container.debug()
    expect(container).toMatchSnapshot()    
})

it('renders Driving Park test', () =>{
    const { getByText } = render(
        <BrowserRouter>
            <Header />
        </BrowserRouter>
    )
    expect(getByText("Driving Park")).toBeInTheDocument();
})

