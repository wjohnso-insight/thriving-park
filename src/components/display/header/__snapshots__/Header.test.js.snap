// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="mobile"
        id="header-wrapper"
      >
        <section
          class="mobile"
          id="header-logo"
        >
          <div
            id="texts-wrapper"
          >
            <div
              id="graflogo-wrapper"
            >
              <h1
                id="sign"
              >
                Driving Park
              </h1>
              <span
                id="graf"
              >
                THRIVING
              </span>
            </div>
            <h4
              class="mobile"
            >
              Always on the move
            </h4>
          </div>
          <div
            class="mobile"
            id="icon-wrapper"
          >
            <div
              class="layout"
            >
              <div
                class="icon"
                id="icon-one"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-car-side fa-w-20 fa-2x "
                  data-icon="car-side"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 640 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M544 192h-16L419.22 56.02A64.025 64.025 0 0 0 369.24 32H155.33c-26.17 0-49.7 15.93-59.42 40.23L48 194.26C20.44 201.4 0 226.21 0 256v112c0 8.84 7.16 16 16 16h48c0 53.02 42.98 96 96 96s96-42.98 96-96h128c0 53.02 42.98 96 96 96s96-42.98 96-96h48c8.84 0 16-7.16 16-16v-80c0-53.02-42.98-96-96-96zM160 432c-26.47 0-48-21.53-48-48s21.53-48 48-48 48 21.53 48 48-21.53 48-48 48zm72-240H116.93l38.4-96H232v96zm48 0V96h89.24l76.8 96H280zm200 240c-26.47 0-48-21.53-48-48s21.53-48 48-48 48 21.53 48 48-21.53 48-48 48z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <div
                class="icon"
                id="icon-two"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-horse fa-w-18 fa-2x "
                  data-icon="horse"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 576 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M575.92 76.6c-.01-8.13-3.02-15.87-8.58-21.8-3.78-4.03-8.58-9.12-13.69-14.5 11.06-6.84 19.5-17.49 22.18-30.66C576.85 4.68 572.96 0 567.9 0H447.92c-70.69 0-128 57.31-128 128H160c-28.84 0-54.4 12.98-72 33.11V160c-48.53 0-88 39.47-88 88v56c0 8.84 7.16 16 16 16h16c8.84 0 16-7.16 16-16v-56c0-13.22 6.87-24.39 16.78-31.68-.21 2.58-.78 5.05-.78 7.68 0 27.64 11.84 52.36 30.54 69.88l-25.72 68.6a63.945 63.945 0 0 0-2.16 37.99l24.85 99.41A15.982 15.982 0 0 0 107.02 512h65.96c10.41 0 18.05-9.78 15.52-19.88l-26.31-105.26 23.84-63.59L320 345.6V496c0 8.84 7.16 16 16 16h64c8.84 0 16-7.16 16-16V318.22c19.74-20.19 32-47.75 32-78.22 0-.22-.07-.42-.08-.64V136.89l16 7.11 18.9 37.7c7.45 14.87 25.05 21.55 40.49 15.37l32.55-13.02a31.997 31.997 0 0 0 20.12-29.74l-.06-77.71zm-64 19.4c-8.84 0-16-7.16-16-16s7.16-16 16-16 16 7.16 16 16-7.16 16-16 16z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <div
                class="icon"
                id="icon-three"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-plane fa-w-18 fa-2x "
                  data-icon="plane"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 576 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M480 192H365.71L260.61 8.06A16.014 16.014 0 0 0 246.71 0h-65.5c-10.63 0-18.3 10.17-15.38 20.39L214.86 192H112l-43.2-57.6c-3.02-4.03-7.77-6.4-12.8-6.4H16.01C5.6 128-2.04 137.78.49 147.88L32 256 .49 364.12C-2.04 374.22 5.6 384 16.01 384H56c5.04 0 9.78-2.37 12.8-6.4L112 320h102.86l-49.03 171.6c-2.92 10.22 4.75 20.4 15.38 20.4h65.5c5.74 0 11.04-3.08 13.89-8.06L365.71 320H480c35.35 0 96-28.65 96-64s-60.65-64-96-64z"
                    fill="currentColor"
                  />
                </svg>
              </div>
            </div>
          </div>
        </section>
        <nav
          class="mobile"
          id="header-nav"
        >
          <section
            class="mobile"
            id="linktab-wrapper"
          >
            <span>
              <a
                class="active"
                href="/"
              >
                about
              </a>
            </span>
          </section>
          <section
            class="mobile"
            id="linktab-wrapper"
          >
            <span>
              <a
                class=""
                href="/move"
              >
                move
              </a>
            </span>
          </section>
          <section
            class="mobile"
            id="linktab-wrapper"
          >
            <span>
              <a
                class=""
                href="/litter"
              >
                litter
              </a>
            </span>
          </section>
        </nav>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="mobile"
      id="header-wrapper"
    >
      <section
        class="mobile"
        id="header-logo"
      >
        <div
          id="texts-wrapper"
        >
          <div
            id="graflogo-wrapper"
          >
            <h1
              id="sign"
            >
              Driving Park
            </h1>
            <span
              id="graf"
            >
              THRIVING
            </span>
          </div>
          <h4
            class="mobile"
          >
            Always on the move
          </h4>
        </div>
        <div
          class="mobile"
          id="icon-wrapper"
        >
          <div
            class="layout"
          >
            <div
              class="icon"
              id="icon-one"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-car-side fa-w-20 fa-2x "
                data-icon="car-side"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 640 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M544 192h-16L419.22 56.02A64.025 64.025 0 0 0 369.24 32H155.33c-26.17 0-49.7 15.93-59.42 40.23L48 194.26C20.44 201.4 0 226.21 0 256v112c0 8.84 7.16 16 16 16h48c0 53.02 42.98 96 96 96s96-42.98 96-96h128c0 53.02 42.98 96 96 96s96-42.98 96-96h48c8.84 0 16-7.16 16-16v-80c0-53.02-42.98-96-96-96zM160 432c-26.47 0-48-21.53-48-48s21.53-48 48-48 48 21.53 48 48-21.53 48-48 48zm72-240H116.93l38.4-96H232v96zm48 0V96h89.24l76.8 96H280zm200 240c-26.47 0-48-21.53-48-48s21.53-48 48-48 48 21.53 48 48-21.53 48-48 48z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <div
              class="icon"
              id="icon-two"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-horse fa-w-18 fa-2x "
                data-icon="horse"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 576 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M575.92 76.6c-.01-8.13-3.02-15.87-8.58-21.8-3.78-4.03-8.58-9.12-13.69-14.5 11.06-6.84 19.5-17.49 22.18-30.66C576.85 4.68 572.96 0 567.9 0H447.92c-70.69 0-128 57.31-128 128H160c-28.84 0-54.4 12.98-72 33.11V160c-48.53 0-88 39.47-88 88v56c0 8.84 7.16 16 16 16h16c8.84 0 16-7.16 16-16v-56c0-13.22 6.87-24.39 16.78-31.68-.21 2.58-.78 5.05-.78 7.68 0 27.64 11.84 52.36 30.54 69.88l-25.72 68.6a63.945 63.945 0 0 0-2.16 37.99l24.85 99.41A15.982 15.982 0 0 0 107.02 512h65.96c10.41 0 18.05-9.78 15.52-19.88l-26.31-105.26 23.84-63.59L320 345.6V496c0 8.84 7.16 16 16 16h64c8.84 0 16-7.16 16-16V318.22c19.74-20.19 32-47.75 32-78.22 0-.22-.07-.42-.08-.64V136.89l16 7.11 18.9 37.7c7.45 14.87 25.05 21.55 40.49 15.37l32.55-13.02a31.997 31.997 0 0 0 20.12-29.74l-.06-77.71zm-64 19.4c-8.84 0-16-7.16-16-16s7.16-16 16-16 16 7.16 16 16-7.16 16-16 16z"
                  fill="currentColor"
                />
              </svg>
            </div>
            <div
              class="icon"
              id="icon-three"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-plane fa-w-18 fa-2x "
                data-icon="plane"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 576 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M480 192H365.71L260.61 8.06A16.014 16.014 0 0 0 246.71 0h-65.5c-10.63 0-18.3 10.17-15.38 20.39L214.86 192H112l-43.2-57.6c-3.02-4.03-7.77-6.4-12.8-6.4H16.01C5.6 128-2.04 137.78.49 147.88L32 256 .49 364.12C-2.04 374.22 5.6 384 16.01 384H56c5.04 0 9.78-2.37 12.8-6.4L112 320h102.86l-49.03 171.6c-2.92 10.22 4.75 20.4 15.38 20.4h65.5c5.74 0 11.04-3.08 13.89-8.06L365.71 320H480c35.35 0 96-28.65 96-64s-60.65-64-96-64z"
                  fill="currentColor"
                />
              </svg>
            </div>
          </div>
        </div>
      </section>
      <nav
        class="mobile"
        id="header-nav"
      >
        <section
          class="mobile"
          id="linktab-wrapper"
        >
          <span>
            <a
              class="active"
              href="/"
            >
              about
            </a>
          </span>
        </section>
        <section
          class="mobile"
          id="linktab-wrapper"
        >
          <span>
            <a
              class=""
              href="/move"
            >
              move
            </a>
          </span>
        </section>
        <section
          class="mobile"
          id="linktab-wrapper"
        >
          <span>
            <a
              class=""
              href="/litter"
            >
              litter
            </a>
          </span>
        </section>
      </nav>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
